submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,174 @@
+
+```
+Setup Costs For First 40MW Capacity, without modules
+| Technology     | Coal Burner |    Solar    |   Nuclear   |
+|:--------------:|:-----------:|:-----------:|:-----------:|
+| Overhead (MW)* |           x |           0 |           z |
+| Iron Plt.      |           x |           y |           z |
+| Copper Plt.    |           x |           y |           z |
+| Steel Plt.     |           x |           y |           z |
+| Coal**         |           0 |           0 |           z |
+| Stone br.      |           0 |           0 |           z |
+| Space (chn.)***|           x |           y |           z |
+*Overhead includes power used by mining drills, assembling machines, centrifuges without to sustain 40MW. Added costs to compensate for overhead are given in parantheses.
+**Used in plastic production for advanced circuits
+***Space Includes chunks used for power production and fuel production areas but not mines. 
+
+Fuel Costs Per Hour for First 40MW Capacity, without modules
+| Technology    | Coal Burner |    Solar    |   Nuclear   |
+|:-------------:|:-----------:|:-----------:|:-----------:|
+| Overhead (MW)*|           x |           0 |           z |
+| Coal          |           x |           0 |           z |
+| Iron Plt.**   |           x |           0 |           z |
+| Uranium Ore   |           x |           0 |           z |
+*Overhead includes power used by mining drills, assembling machines, centrifuges to sustain 40MW. Added costs to compensate for overhead are given in parantheses.
+**Inlcudes iron plates used to produce sulfuric acid for mining and for uranium fuel cells.
+```
+
+* **Nuclear reactors are far too expensive to maintain without the Kovarex Enrichment Process** – Obtaining one piece of Uranium-235 indeed requires centrifuging thousands of uranium ore pieces, but on the other hand 1 piece of it is enough to make 10 uranium fuel cells, which last over 30 minutes in total. In a rather eye-opening [video](https://youtu.be/Qw_NzPuccxk?t=1610) Nilaus walks us through the mathematics and from there we see that you need only 8-9 electric miners and 1 centrifuge to maintain 1 nuclear reactor continuously without the Kovarex Enrichment Process, although it does not hurt to process a little more in preparation for unlocking Kovarex or in case of bad luck during centrifuging.
+* **Nuclear power is a big polluter** - In vanilla Factorio, nuclear radiation causes no damage and has no effects on biter evolution, while regular pollution does. The pollution caused by nuclear power is from mining and processing uranium ore, but it is substantially small in terms of pollution per MW when compared to coal.  <!--If time left, I will calculate this one too.-->

```

Comment:
if you address the pollution point, you have to do the calculations. If no calculations, then the point have to be removed.

Reply:
Yeah might as well